### PR TITLE
Token support UI fixes

### LINKED ISSUE #1455: Increase stroke for icons to 1.25
See details [here](https://www.notion.so/valora-inc/Multi-Token-Support-Design-QA-d836e15ac5ab4bf7a39ad0f3d8ca59cf)

AC
* Stroke is increased to 1.25

### PR DESCRIPTION
**Pull request by gnardini**
### Description

A few fixes:
- Fix the stroke width prop of SVGs.
- Don't show the token symbol twice on the token balances screen.
- Make the dismiss button in the `NotificationBox` `gray4`

### Other changes

- Remove dark mode code since it's not on-brand and won't be used as-is anymore.

### Tested

Manually and updated unit tests.

### How others should test

Check the token balances screen and icons and see that they all look good.

### Related issues

- Fixes #1455

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/icons/DeniedIcon.tsx [hunk 1]
```diff
@@ -10,18 +10,18 @@ interface Props {
...
         stroke-linecap="round"
         stroke-linejoin="round"
       />
       <Path
         d="M11 11L5.00002 5.00002"
         stroke={color}
-        stroke-width="1.5"
+        strokeWidth="1.5"
         stroke-linecap="round"
         stroke-linejoin="round"
```
Comment (jeanregisser) [inline packages/mobile/src/icons/DeniedIcon.tsx:26]: Are these supported?

**Review by jeanregisser — APPROVED** [review]
Great! 👍

File: packages/mobile/src/icons/DeniedIcon.tsx (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/icons/DeniedIcon.tsx:26]: They are but with a different name (using camelCase), I'll update those as well

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this issue using Android internal build and iOS test flight build V:1.22.1 & updated observation in Gsheet
**Gsheet Link:**
https://docs.google.com/spreadsheets/d/1UVmYHl_g7ZWKqX3zBtc5OdO9wEmdAdj4PW9zgwJxUys/edit#gid=726255110
CC: @gnardini 
Thanks..!!

**Pull request merged**